src/js/sdk.tsx

```diff
@@ -44,6 +44,10 @@ const DEFAULT_OPTIONS: ReactNativeOptions = {
  * Inits the SDK and returns the final options.
  */
 export function init(passedOptions: ReactNativeOptions): void {
+  if (isRunningInMetroDevServer()) {
```

Comment:
Maybe I should name the function better, I believe there is no case at the moment when users would want the SDK to run in this case.

At this happens only during the bundling process when metro generates the static routes for the web bundles.

I didn't add any log as I want to avoid confusion, seeing `Sentry not initialized` and after it `Sentry initialized`, both the bundle process and the JS logs are in one output.

Reply:
I see, then this will not be a problem.